### DISCUSSION TITLE
Support for "not equal" queries in Firestore

### DISCUSSION THREAD

**Discussion by choibles** [invertase/react-native-firebase#4406] [Ideas] [answered]
Hi – first off, thanks for the great library!

Wondering if there's any plans to support the new "not equal" operator for Firestore queries. The [Firebase blog](https://firebase.googleblog.com/2020/09/cloud-firestore-not-equal-queries.html) indicated that as of this month (Oct 2020) Firestore now supports them.

Not an urgent request (know it just went up 2 weeks ago) but would definitely be a welcome addition to the library. Thanks in advance.

**Comment by mikehardy** [comment, marked as answer]
dupe of https://github.com/invertase/react-native-firebase/discussions/4399

**Reply by mikehardy** [reply]
Will be supported in #4474
